protocol/discord/assets.go

```diff
@@ -0,0 +1,35 @@
+package discord
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/status-im/status-go/images"
+)
+
+func DownloadAvatarAsset(url string) ([]byte, error) {
+	imgs, err := images.GenerateIdentityImagesFromURL(url)
+	if err != nil {
+		return nil, err
+	}
+	payload := imgs[0].Payload
+	return payload, nil
+}
+
+func DownloadAsset(url string) ([]byte, string, error) {
+	client := http.Client{}
```

Comment:
Should the httpClient have a timeout?
```suggestion
	client := http.Client{Timeout: time.Minute} // 1 min

```